scripts/megamind-healthcheck.mjs

```
@@ -0,0 +1,65 @@
+  runtime_tested: false,
+  next_commands: [
+    "yarn install --frozen-lockfile",
+    "yarn build",
+    "yarn test",
+    "yarn cli --help",
+  ],
+  truth_note:
+    "This check does not install packages, read environment secrets, open a browser, contact a provider, or execute a web mission.",
+};
+
+console.log(JSON.stringify(report, null, 2));
```

ESLint

[error] 'console' is not defined.

(no-undef)